doc/Streaming.xml

```diff
@@ -590,7 +605,245 @@
         </section>
         <section>
           <title>SRTP data transfer via UDP</title>
-          <para>This mode allows secure transmission of RTP packets via UDP unicast and multicast. See RFC 3711 for transmission and RFC 4567 for key exchange.</para>
+            <para>This mode allows secure transmission of RTP packets via UDP unicast and multicast. Related documents are RFC 3711, RFC 7714 for transmission and RFC 3830, RFC 4567 for key exchange.</para>
+            <para>
+            The device shall list whether or not it supports SRTP streaming through the GetServiceCapabilities action's response. If the device supports SRTP streaming, the device shall indicate it through the SecureRTSPStreaming parameter in the StreamingCapabilities.
+          </para>
+          <section>
+            <title>Cryptographic algorithm negotiation</title>
+            <para>
+              The device may list which SRTP cryptographic algorithms it supports through the Media2 wsdl.
+              Each encoder can list the algorithms it supports through the SecureStreamingProtocolAlgorithms element present in:
```

Comment:
This line can then be deleted as we made the requirement text compact as suggested/proposed above.

```suggestion
```